spec/index.bs

```diff
@@ -1258,11 +1258,22 @@ Note: This also applies to events in [[SVG2#EventAttributes]].
 ### HostEnsureCanCompileStrings ### {#host-ensure-can-compile-strings}
 
 JavaScript contains an <span>implementation-defined</span> <a href="https://tc39.es/ecma262/#sec-hostensurecancompilestrings">HostEnsureCanCompileStrings</a>(<var>realm</var>, <ins><var>parameterStrings</var>,
-<var>bodyString</var>, <var>codeString</var>, <var>compilationType</var>, <var>parameterArgs</var>, <var>bodyArg</var></ins>)
+<var>bodyString</var>, <var>compilationType</var>, <var>parameterArgs</var>, <var>bodyArg</var></ins>)
 abstract operation. User agents must use the following implementation:
 
+Issue: ECMA262 doesn't pass a constructed code string through so we have to reconstruct it ourselves. This needs to be defined.
+
+1. Let |source| be the result of reconstructing the code string from |parameterStrings|, |bodyString|, |compilationType|.
```

Comment:
Need to actually define this. CSP currently expects a full code string so I think we have to spec that, shouldn't be too bad to do. Not sure if this is best done in HTML or CSP, probably CSP?